xsd/netex_framework/netex_responsibility/netex_entity_support.xsd

```diff
@@ -118,19 +118,23 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Reference to a Type of an ENTITY.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:complexType name="ClassRefStructure">
+	<xsd:complexType name="ClassRefStructure" mixed="true">
```

Comment:
What do you want to achieve with mixed="true" ? It's usually to mix text and element (for example "Dear Mr. <name>John Smith</name>") ...